libtenzir/builtins/contexts/lookup-table.cpp

```diff
@@ -125,6 +125,28 @@ class ctx final : public virtual context {
     return record{{"num_entries", context_entries.size()}};
   }
 
+  auto dump_data_entries() const -> generator<table_slice> {
+    auto entry_builder = series_builder{};
+    for (const auto& [key, value] : context_entries) {
+      entry_builder.data(record{{"key", key}, {"value", value}});
+      auto slices = entry_builder.finish_as_table_slice();
+      for (auto&& slice : slices) {
+        co_yield slice;
+      }
+    }
+  }
+
+  auto dump() -> caf::expected<std::vector<table_slice>> override {
+    if (not entry_dumper) {
+      entry_dumper = dump_data_entries();
+    }
+    for (const auto&& x : *entry_dumper) {
+      return std::vector<table_slice>{x};
+    }
+    entry_dumper.reset();
+    return std::vector<table_slice>{};
+  }
```

Comment:
I don't understand what this function does.

Reply:
This will be gone.